### ISSUE TITLE
DataTable OverflowMenu has DAP Violations

### ISSUE THREAD

**Issue by megbridge** [carbon-design-system/carbon#3673]
## Environment

> Operating system
All

> Browser
Chrome 

> Automated testing tool and ruleset
DAP IBM Accessibility August 2018

## Detailed description

> What version of the Carbon Design System are you using?
v10.2

> What WCAG 2.1 checkpoint does the issue violate?
"All content must reside within a WAI-ARIA landmark or labelled region role"

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
https://codesandbox.io/embed/datatable-j1o0f

**Comment by tw15egan** [comment]
@dakahn Since the Overflow menu get's placed adjacent to the root react div, what should it's role be? The `main` element is inside that root element, so the overflow menu `ul` is just hanging out in no man's land

**Comment by abbeyhrt** [comment]
Hi @megbridge! Could you reproduce the violation with the updated versions of carbon-components and carbon-components-react? I'm unable to find any violations specifically around the OverflowMenu in DataTable with the newest versions. A screenshot of the DAP violation would be super helpful if you can!

**Comment by megbridge** [comment]
Hi Abbey, it looks like the specific violation I pointed out above isn't present with the most recent versions of carbon-components and carbon-components-react. However, I still see DAP violations with the table.  

https://codesandbox.io/embed/datatable-k8e59

![Screen Shot 2019-09-11 at 2 01 05 PM](https://user-images.githubusercontent.com/20115228/64722605-cfee5300-d49c-11e9-8cac-50eba26a9283.png)

**Comment by abbeyhrt** [comment]
Oh for sure, DataTable needs a ton of work a11y-wise that we're chipping away at. I just wanted to make sure that the issues specifically around the OverflowMenu had been addressed. Thank you!

**Comment by abbeyhrt** [comment]
Closing this since it has been addressed and #3673 logs the other DataTable violations.
